src/commands/app/scaffold.ts

```diff
@@ -35,23 +35,23 @@ export default class AppScaffold extends Kommand {
 
     const tasks = new Listr([
       {
-        title: `Creating ${chalk.gray(this.args.name + path.sep)} directory`,
+        title: `Creating directory: ${chalk.gray(this.args.name + path.sep)}`,
         task: () => execute('mkdir', this.args.name)
       },
       {
         title: 'Creating and rendering application files',
         task: () => this.renderTemplates(templatePath, templatePath)
       },
       {
-        title: 'Installing latest Kuzzle version via NPM and Docker (this can take some time)',
+        title: 'Installing dependencies (this can take some time)',
         task: () =>
-          execute('npm', 'run', 'npm:docker', 'install', 'kuzzle', { cwd: this.args.name })
+          execute('npm', 'run', 'install:docker', { cwd: this.args.name })
```

Comment:
But then here `kuzzle` is not installed, the dependency is not listed in the scaffolded `package.json` since we want to always install the latest version

Reply:
We should first run `npm run install:docker` to install dependencies and then run `npm run npm:docker install kuzzle` to install Kuzzle latest version

Reply:
I want to check something first... I think that we can simply specify a major version for a NPM dependency. If we can do that, then we can let NPM do the job for us by simpling configuring the package.json file differently.

Reply:
@Aschen > the last commit should make sure that NPM installs the latest kuzzle v2 version when scaffolding a new application. Can you check it out?